UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -35,10 +35,16 @@ public class UndertaleEmbeddedTexture : UndertaleNamedResource, IDisposable
     public uint GeneratedMips { get; set; }
 
     /// <summary>
-    /// TODO: something. <br/>
+    /// The size of the texture data in the embedded image. <br/>
     /// GameMaker: Studio 2 only.
     /// </summary>
-    public uint TextureBlockSize { get; set; }
+    private uint _TextureBlockSize { get; set; }
+
+    /// <summary>
+    /// The position of the TextureBlockSize value to be overwritten in Serialize. <br/>
+    /// Only used internally in UMTLib for GM 2022.3 support.
+    /// </summary>
+    internal uint _TextureBlockSizeLocation { get; set; }
```

Comment:
If it's only used internally, change it to `private` please.

Reply:
```suggestion
    private uint _TextureBlockSizeLocation { get; set; }
```